src/main/java/frc/robot/subsystems/ClimbSubsystem.java

```diff
@@ -103,409 +106,276 @@
-      sbHaHeightOne.setNumber(HaHeightOne);
+      poleOne.getPIDCtrl().setReference(poleHeightOne, CANSparkMax.ControlType.kPosition);
+      sbpoleHeightOne.setNumber(poleHeightOne);
 
-      m_HaTwo.getPIDCtrl().setReference(HaHeightOne, CANSparkMax.ControlType.kPosition);
-      sbHaHeightTwo.setNumber(HaHeightOne);
+      poleTwo.getPIDCtrl().setReference(poleHeightOne, CANSparkMax.ControlType.kPosition);
+      sbpoleHeightTwo.setNumber(poleHeightOne);
     }
   }
 
-  public void deployHA() {
-    HaHeightOne = Constants.HaHeightMax;
-    HaHeightTwo = Constants.HaHeightMax;
-    m_HaOne.getPIDCtrl().setReference(HaHeightOne, CANSparkMax.ControlType.kPosition);
-    sbHaHeightOne.setNumber(HaHeightOne);
-
-    m_HaTwo.getPIDCtrl().setReference(HaHeightOne, CANSparkMax.ControlType.kPosition);
-    sbHaHeightTwo.setNumber(HaHeightOne);
+  // called at the start of auton
+  public void retractPoles() {
+    poleOne.getPIDCtrl().setReference(Constants.poleHeightMax, CANSparkMax.ControlType.kPosition);
+    poleTwo.getPIDCtrl().setReference(Constants.poleHeightMax, CANSparkMax.ControlType.kPosition);
   }
 
-  public void periodic() {
-    if (DriverStation.isDisabled() && climbEnabble) {
-      climbEnable();
-    }
-
-    if (sbMcHeightOne.getDouble(0) != McHeightOne) {
-      McHeightOne = sbMcHeightOne.getDouble(0);
-    }
-
-    if (sbMcHeightTwo.getDouble(0) != McHeightTwo) {
-      McHeightTwo = sbMcHeightTwo.getDouble(0);
-    }
-
-    BClimbEnabled.setBoolean(climbEnabble);
-    DClimbHeight1.setNumber(McHeightTwo);
-    DClimbHeight2.setNumber(HaHeightTwo);
-  }
-
-  public void debugPeriodic() {
-    if (DriverStation.isDisabled() && climbEnabble) {
-      climbEnable();
-    }
-
-    SmartDashboard.putNumber("Mc1 Applied Output", m_McOne.getAppliedOutput());
-    SmartDashboard.putNumber("Mc1 Hight", m_McOne.getEncoder().getPosition());
-    SmartDashboard.putNumber("Mc1 IAccum", m_McOne.getPIDCtrl().getIAccum());
-
-    SmartDashboard.putNumber("Mc2 Applied Output", m_McTwo.getAppliedOutput());
-    SmartDashboard.putNumber("Mc2 Hight", m_McTwo.getEncoder().getPosition());
-    SmartDashboard.putNumber("Mc2 IAccum", m_McTwo.getPIDCtrl().getIAccum());
-
-    SmartDashboard.putNumber("Ha1 Applied Output", m_HaOne.getAppliedOutput());
-    SmartDashboard.putNumber("Ha1 Hight", m_HaOne.getEncoder().getPosition());
-    SmartDashboard.putNumber("Ha1 IAccum", m_HaOne.getPIDCtrl().getIAccum());
-
-    SmartDashboard.putNumber("Ha2 Applied Output", m_HaTwo.getAppliedOutput());
-    SmartDashboard.putNumber("Ha2 Hight", m_HaTwo.getEncoder().getPosition());
-    SmartDashboard.putNumber("Ha2 IAccum", m_HaTwo.getPIDCtrl().getIAccum());
-
-    // Mid Climb
-    if (sbMcHeightOne.getDouble(0) != McHeightOne) {
-      McHeightOne = sbMcHeightOne.getDouble(0);
+  // functions for ClimbSequence1
+  public void deployArms() {
+    if (armEncoderHeightOne < Constants.armHeightFeather
+        && armEncoderHeightTwo < Constants.armHeightFeather) {
```

Comment:
OR?

Problem you are trying to automate is that the initial movement of the arms needs to be slow.  If one of the arms is below that minimum safe point, you should probably continue using the velocity PID.